yogstation/code/modules/holidays/holidays.dm

```diff
@@ -47,12 +47,6 @@
 		"https://www.youtube.com/watch?v=PJR3xTdbXH8" // Nyet Molotov
 		)
 
-/datum/holiday/oakday/getStationPrefix()
```

Comment:
/datum/holiday/gondoladay/getStationPrefix()
	return pick("Gondola","Finnish","Finland","Karelia","Perkele")
	
/datum/holiday/gondoladay/greet()
	return "Happy Gondola day!"

Reply:
I don't know how to make it suggest code changes but ye